watchtower-plugin/requirements.txt

```diff
@@ -1,8 +1,8 @@
 pyln-client
 requests
 coincurve
-cryptography==2.8
+cryptography>=2.8
```

Comment:
I assume the vulnerability mentioned is the one addressed in `3.2`? If so, shouldn't this be `>=3.2` as the previous 2.8 still satifies this

Reply:
Hey @Legogris, thanks for the ping about this.

That is indeed the vulnerability mentioned. I think I set it to `>=` so it gets the latest version, but didn't really bump the minimum since the vulnerability seems to be rather tangential to our use of the library. We only use `cryptography` for `ChaCha20Poly1305`.

However, this made me realize that the requirements are actually outdated, since the only module that currently needs the `cryptography` module is `common`. I'll double check that everything works as expected with 3.2 and update all requirement files.